Assets/Scripts/Crew.cs

```diff
@@ -24,30 +26,32 @@ public class Crew : MonoBehaviour
 
 	public void Awake()
 	{
-		randomMaleNameButton.onClick.AddListener(delegate { randomPilotName("male"); });
-		randomFemaleNameButton.onClick.AddListener(delegate { randomPilotName("female"); });
-		randomRobotNameButton.onClick.AddListener(delegate { randomPilotName("robot"); });
+		maleNames = PilotNameDataSingleton.Instance.MaleNames;
+		femaleNames = PilotNameDataSingleton.Instance.FemaleNames;
+
+		randomMaleNameButton.onClick.AddListener(delegate { randomPilotName(Gender.male); });
```

Comment:
to make sure you cant add two listeners on accident, always call this first: Button.onClick.RemoveAllListeners()